content/src/content/jcr_root/apps/core/wcm/components/commons/v1/templates.html

```diff
@@ -18,4 +18,10 @@
     <div data-sly-test="${(wcmmode.edit || wcmmode.preview) && isEmpty}"
          class="cq-placeholder ${classAppend}"
          data-emptytext="${component.properties.jcr:title}${emptyTextAppend && ' - '}${emptyTextAppend}"></div>
-</sly>
\ No newline at end of file
+</sly>
+
+<sly data-sly-template.hiddenFieldPlaceholder="${@ isEmpty, classAppend, id, name, value}">
```

Comment:
If a hidden field is always empty, should we also remove `isEmpty` parameter?

Reply:
I think I would prefer if we just use the current placeholder template and make use of the `emptyTextAppend` parameter for building and adding this string in the hidden field itself.

Reply:
This has been addressed, please see comment above.